crates/wasi-provider/src/wasi_runtime.rs

```diff
@@ -242,13 +255,18 @@ impl WasiRuntime {
                     timestamp: chrono::Utc::now(),
                 })
                 .expect("status should be able to send");
-            match instance
-                .get_export("_start")
-                .expect("_start import should exist in wasm module")
-                .func()
-                .unwrap()
-                .call(&[])
-            {
+            let export = instance.get_export("_start").ok_or(anyhow::anyhow!(
```

Comment:
Perhaps `ok_or_else` rather than `ok_or`?  Depending on the features used, `anyhow!` can do some expensive things such as capture a backtrace or format error messages.

Reply:
Good point. I'll switch that out